### PR TITLE
Nodes Clusters docs fixes during ROSA review

### PR DESCRIPTION
**Pull request by mburke5678**
Fixing various errors in the Nodes -> Cluster docs as I find them during the ROSA content port. Mostly formatting, wording, and such.

[Supported TuneD daemon plugins](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-node-tuning-operator#supported-tuned-daemon-plug-ins_nodes-node-tuning-operator) - Changed links to Additional references ([current docs](https://docs.openshift.com/container-platform/4.13/nodes/nodes/nodes-node-tuning-operator.html#supported-tuned-daemon-plug-ins_nodes-node-tuning-operator))
[Creating a Limit Range](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-limit-ranges#nodes-cluster-limit-creating_nodes-cluster-limit-ranges) -- Added `[source,terminal]` to step 2.
[Deleting a Limit Range](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-limit-ranges#nodes-cluster-limit-ranges-deleting_nodes-cluster-limit-ranges) -- Added `[source,terminal]` to step 2.
[Viewing a limit](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-limit-ranges#nodes-cluster-limit-viewing_nodes-cluster-limit-ranges) -- Added `[source,terminal]` throughout
[Disabling or enforcing CPU limits using CPU CFS quotas](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-overcommit#nodes-cluster-overcommit-node-enforcing_nodes-cluster-overcommit) -- Added `[source,terminal]` to prereq tip and made prereq a bullet, not a number ([current docs](https://docs.openshift.com/container-platform/4.13/nodes/clusters/nodes-cluster-overcommit.html#nodes-cluster-overcommit-node-enforcing_nodes-cluster-overcommit))
[Disabling overcommitment for a project](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-overcommit#nodes-cluster-overcommit-project-disable_nodes-cluster-overcommit) -- Added proper example code.
[Understanding how to override the JVM maximum heap size](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-resource-configure#nodes-cluster-resource-configure-jdk-heap_nodes-cluster-resource-configure) -- Changed numbered list to unordered list.
[Understanding OOM kill policy](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-resource-configure#nodes-cluster-resource-configure-oom_nodes-cluster-resource-configure) -- Split the command and output.
[Finding the memory request and limit from within a pod](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-resource-configure#nodes-cluster-resource-configure-request-limit_nodes-cluster-resource-configure) -- Added supsteps and verification. ([current docs](https://docs.openshift.com/container-platform/4.13/nodes/clusters/nodes-cluster-resource-configure.html#nodes-cluster-resource-configure-request-limit_nodes-cluster-resource-configure))
[Configuring cluster-level overcommit](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-overcommit#nodes-cluster-resource-configure_nodes-cluster-overcommit) -- Added # ...
[Understanding the OpenShift Cluster Capacity Tool](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-resource-levels#nodes-cluster-resource-levels-about_nodes-cluster-resource-levels) -- Updated the name of the cluster capacity tool, based on [the source](https://catalog.redhat.com/software/containers/openshift4/ose-cluster-capacity/5cca0324d70cc57c44ae8eb6?container-tabs=overview). ([current docs](https://docs.openshift.com/container-platform/4.13/nodes/clusters/nodes-cluster-resource-levels.html#nodes-cluster-resource-levels-about_nodes-cluster-resource-levels))
[Running the OpenShift Cluster Capacity Tool on the command line](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-resource-levels#nodes-cluster-resource-levels-command_nodes-cluster-resource-levels) -- Cleaned up the procedure. ([current docs](https://docs.openshift.com/container-platform/4.13/nodes/clusters/nodes-cluster-resource-levels.html#nodes-cluster-resource-levels-command_nodes-cluster-resource-levels))
[Running the OpenShift Cluster Capacity Tool as a job inside a pod](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-resource-levels#nodes-cluster-resource-levels-job_nodes-cluster-resource-levels) -- Cleaned up the procedure. Changed link to OpenShift repo ([current docs](https://docs.openshift.com/container-platform/4.13/nodes/clusters/nodes-cluster-resource-levels.html#nodes-cluster-resource-levels-job_nodes-cluster-resource-levels))
[Installing the Cluster Resource Override Operator using the CLI](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-overcommit#nodes-cluster-resource-override-deploy-cli_nodes-cluster-overcommit) -- Changed ... to # ... in Step 5.
[Installing the Cluster Resource Override Operator using the web console](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-overcommit#nodes-cluster-resource-override-deploy-console_nodes-cluster-overcommit) -- Cleaned up the procedure.
[Cluster-level overcommit using the Cluster Resource Override Operator](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-overcommit#nodes-cluster-resource-override_nodes-cluster-overcommit) -- Added # ...
[Estimating the number of pods your {product-title} nodes can hold](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-resource-levels) -- Updated the name of the cluster capacity tool
[Machine Config Daemon metrics](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-machine-config-daemon-metrics) -- Changed wording of link text
[Configuring an OpenShift Container Platform cluster for pods](https://63085--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-pods-configuring.html) -- Added assembly that was created for the 3.x -> 4.x conversion, but never added.

### CODE REVIEW

File: modules/nodes-cluster-resource-levels-command.adoc [hunk 1]
```diff
@@ -4,20 +4,22 @@
...
-= Running the cluster capacity tool on the command line
+= Running the OpenShift Cluster Capacity Tool on the command line
 
-You can run the {product-title} cluster capacity tool from the command line
+You can run the OpenShift Cluster Capacity Tool from the command line
 to estimate the number of pods that can be scheduled onto your cluster.
 
+You create a sample pod spec file, which the tool uses for estimating resource usage. The pod spec specifies its resource
+requirements as `limits` or `requests`. The cluster capacity tool takes the
+pod's resource requirements into account for its estimation analysis.
```
Comment (mburke5678) [inline modules/nodes-cluster-resource-levels-command.adoc:14]: [Existing text moved](https://github.com/openshift/openshift-docs/pull/63085/files#diff-9a6729224de7e14a526bdbeae13a8ce30330160e7bb6bf9a50fa0e7d37374a35L16) out of procedure.

**Review by mjpytlak — COMMENTED** [review]
@mburke5678 Just some odds and ends to address.

File: modules/nodes-cluster-resource-levels-about.adoc [hunk 2]
```diff
@@ -4,9 +4,9 @@
 
 :_content-type: CONCEPT
 [id="nodes-cluster-resource-levels-about_{context}"]
-= Understanding the {product-title} cluster capacity tool 
+= Understanding the OpenShift Cluster Capacity Tool
 
-The cluster capacity tool simulates a sequence of scheduling decisions to
+The OpenShift Cluster Capacity Tool simulates a sequence of scheduling decisions to
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/nodes-cluster-resource-levels-about.adoc:9]: @mjpytlak I left a bunch of hard wrapped lines as is, to focus the reviewer's attention on the "real" changes. Maybe that is bad form.

File: modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc [hunk 3]
```diff
@@ -38,8 +39,8 @@ that is not supported. The following TuneD plugins are currently not supported:
...
-See
-link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/monitoring_and_managing_system_status_and_performance/customizing-tuned-profiles_monitoring-and-managing-system-status-and-performance#available-tuned-plug-ins_customizing-tuned-profiles[Available
-TuneD Plugins] and
-link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/monitoring_and_managing_system_status_and_performance/getting-started-with-tuned_monitoring-and-managing-system-status-and-performance[Getting
-Started with TuneD] for more information.
+.Additional references
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/monitoring_and_managing_system_status_and_performance/customizing-tuned-profiles_monitoring-and-managing-system-status-and-performance#available-tuned-plug-ins_customizing-tuned-profiles[Available TuneD Plugins]
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/monitoring_and_managing_system_status_and_performance/getting-started-with-tuned_monitoring-and-managing-system-status-and-performance[Getting Started with TuneD]
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc:46]: @mjpytlak Good catch. I opened an [issue to address this](https://issues.redhat.com/browse/OSDOCS-7447), as my PR is 4.10+.

**Pull request merged**